apps/governance/src/lib/web3-connectors.ts

```diff
@@ -20,23 +20,27 @@ export const createDefaultProvider = (providerUrl: string, chainId: number) => {
   return new ethers.providers.JsonRpcProvider(providerUrl, chainId);
 };
 
+const [metamask, metamaskHooks] = initializeMetaMaskConnector();
+
 export const createConnectors = (providerUrl: string, chainId: number) => {
   if (isNaN(chainId)) {
     throw new Error('Invalid Ethereum chain ID for environment');
   }
-  const [walletconnect, walletconnectHooks] =
-    initializeConnector<WalletConnect>(
-      (actions) =>
-        new WalletConnect(actions, {
-          rpc: {
-            [chainId]: providerUrl,
-          },
-        }),
-      [chainId]
-    );
+  const [coinbase, coinbaseHooks] = initializeCoinbaseConnector(providerUrl);
+  const [walletconnect, walletconnectHooks] = initializeWalletConnector(
+    chainId,
+    providerUrl
+  );
+  const [legacy, legacyHooks] = initializeWalletConnectLegacyConnector(
```

Comment:
suggestion (non blocking): I think a more explicity name here would be helpful perhaps `walletconnectLegacy` or `walletconnectV1`